ArduPlane/Log.cpp

```diff
@@ -505,7 +509,7 @@ void Plane::Log_Read(uint16_t list_entry, int16_t start_page, int16_t end_page)
 
     cliSerial->printf("%s\n", HAL_BOARD_NAME);
 
-	DataFlash.LogReadProcess(list_entry, start_page, end_page,
+   DataFlash.LogReadProcess(list_entry, start_page, end_page,
```

Comment:
> Sorry, I don't know what you mean by commit squashing and tidying (beyond addressing reviewers' comments, including yours); please clarify.

I mean using interactive rebase to cleanup the commits to make them more reviewable and to not include all the changing back and forth. If you haven't done it before I could do it for you.

> In other words, compared to the main ArduPilot branch, there shouldn't be any space changes to the existing code.

There are quite a few actually. This link takes you to the overall effect of all the changes in the PR:
  https://github.com/ArduPilot/ardupilot/pull/5737/files
you can see quite a few whitespace changes to unrelated files.